pt-BR/getting_started.md

```diff
@@ -582,30 +582,32 @@ This action is now displaying the parameters for the article that are coming in
 from the form. However, this isn't really all that helpful. Yes, you can see the
 parameters but nothing in particular is being done with them.
 
-### Creating the Article model
+### Criando um model para o `Article` (artigo, inglês)
```

Comment:
```suggestion
### Criando um *model* para o `Article` (artigo, inglês)
```
Não estamos traduzindo alguns termos técnicos, como model, view etc. Só colocando itálico. 
Estamos sempre atualizando o manual de tradução do Guia: https://github.com/campuscode/rails-guides-pt-BR/blob/master/TRANSLATION_MANUAL.md